# Logitech F710
axis_linear:
  x: 1
scale_linear:
  x: 0.7
axis_angular:
  yaw: 3
scale_angular:
  yaw: 0.7

start_button: 7  # Trigger start
enable_driving: 5 # must continuing press this button to pub vel
enable_turbo_button: -1  # Button 2 aka thumb button

debug_print: true